lms/static/js/student_account/views/RegisterView.js

```diff
@@ -239,7 +239,6 @@
                         };
 
                     FormView.prototype.postRender.call(this);
-                    $('.optional-fields').addClass('hidden');
```

Comment:
Can we please some how move this change into our registration js file, and revert the core change?

Reply:
I have tried to do this in our changes. But issue here is the this code is executed after our code and till that optional-fields do not contains hidden class so can't remove this class using .removeClass, Will need to do this changes. Can't avoid this.

Reply:
Actually I tried this one. We need this core change as this code is executed after our code and that is why I can't do .removeClass('hidden') in our code.